hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/capacity/CapacitySchedulerTestBase.java

```diff
@@ -27,37 +27,14 @@
 import java.util.Set;
 
 public class CapacitySchedulerTestBase {
-  protected final int GB = 1024;
-
-  protected static final String A = CapacitySchedulerConfiguration.ROOT + ".a";
-  protected static final String B = CapacitySchedulerConfiguration.ROOT + ".b";
-  protected static final String A1 = A + ".a1";
-  protected static final String A2 = A + ".a2";
-  protected static final String B1 = B + ".b1";
-  protected static final String B2 = B + ".b2";
-  protected static final String B3 = B + ".b3";
-  protected static float A_CAPACITY = 10.5f;
-  protected static float B_CAPACITY = 89.5f;
-  protected static final String P1 = CapacitySchedulerConfiguration.ROOT + ".p1";
-  protected static final String P2 = CapacitySchedulerConfiguration.ROOT + ".p2";
-  protected static final String X1 = P1 + ".x1";
-  protected static final String X2 = P1 + ".x2";
-  protected static final String Y1 = P2 + ".y1";
-  protected static final String Y2 = P2 + ".y2";
-  protected static float A1_CAPACITY = 30;
-  protected static float A2_CAPACITY = 70;
-  protected static float B1_CAPACITY = 79.2f;
-  protected static float B2_CAPACITY = 0.8f;
-  protected static float B3_CAPACITY = 20;
-
+  public static final int GB = 1024;
```

Comment:
I think it would be better if these method would go into the helper class and TestCapacityScheduler just had some static imports.

Reply:
There is a separate jira (YARN-10963) which splits the **TestCapacityScheduler** to multiple test files, for that the **CapacitySchedulerTestBase** class makes sense.

Reply:
Made it to a utility class: CapacitySchedulerTestUtilities.